distributed/scheduler.py

```diff
@@ -3752,12 +3752,9 @@ async def add_worker(
 
         version_warning = version_module.error_message(
             version_module.get_versions(),
-            merge(
-                {w: ws.versions for w, ws in self.workers.items()},
-                {c: cs.versions for c, cs in self.clients.items() if cs.versions},
-            ),
+            {w: ws.versions for w, ws in self.workers.items()},
             versions,
-            client_name="This Worker",
+            source_name=f"Current worker: {ws.server_id}",
```

Comment:
Does asciitable render a newline properly? `Current worker:\n{ws.server_id}`
This way we'd reduce horizontal space of the warning by 16 chars

Reply:
Let me take a look. IMO, we could also just skip `Current worker: ` entirely. The source should be sufficiently identified by its ID. Though my guess would be that it wreaks havoc on the formatting since it should break in the middle of the table header.

Reply:
Yup, that just messes things up. I'll remove `Current worker: ` so that the message will look like this:
```
2022-08-10 15:42:31,352 - distributed.worker - WARNING - Mismatched versions found

+---------+---------------------------------------------+----------------------+-----------------------------------+
| Package | Worker-5a4690f1-71ca-4c19-9b3b-22ac5c13674a | Scheduler            | Workers                           |
+---------+---------------------------------------------+----------------------+-----------------------------------+
| dask    | 2022.8.0+5.gf7fd6c48                        | 2022.8.0+5.gf7fd6c48 | {'2022.8.0+5.gf7fd6c48', '0.0.0'} |
+---------+---------------------------------------------+----------------------+-----------------------------------+
```